Source/ProgressionSystemRuntime/Public/LevelActors/PSStarActor.h

```diff
@@ -14,26 +14,50 @@ class PROGRESSIONSYSTEMRUNTIME_API APSStarActor : public AActor
 	// Sets default values for this actor's properties
 	APSStarActor();
 
+	/** Static helper function that plays any given star animation from various places. */
+	UFUNCTION(BlueprintCallable, Category= "C++")
+	bool TryPlayStarAnimation(UPARAM(ref) float& StartTimeRef, UCurveTable* AnimationCurveTable);
+
+	/** Set the start time for hiding stars in the main menu */
+	UFUNCTION(BlueprintCallable, Category = "C++", meta = (BlueprintProtected))
+	void SetStartTimeHideStars();
+
+	/** Set the start time for main menu stars animation */
+	UFUNCTION(BlueprintCallable, Category = "C++", meta = (BlueprintProtected))
```

Comment:
Remove `BlueprintProtected` from both Setters as they are under `public` specifiers already

Reply:
got it ! https://github.com/h4rdmol/ProgressionSystem/pull/14/commits/ff594d89d5377e1c9d03df3a880f6bb009e56af5